### DISCUSSION TITLE
Failure in updating Firedrake

### DISCUSSION THREAD

**Discussion by yianzeng** [firedrakeproject/firedrake#2906] [Firedrake support] [answered]
I get the following error when I run 'firedrake-update' with my venv activated.
Running Python 3.10.8 (via homebrew) on an M1 Mac
```
Running /Users/yianzeng/Desktop/coding/firedrake/bin/firedrake-update
Traceback (most recent call last):
  File "/Users/yianzeng/Desktop/coding/firedrake/bin/firedrake-update", line 436, in <module>
    parser.add_argument("--netgen", action="store_true", dest="netgen", default=config["options"]["netgen"],
KeyError: 'netgen'
```

**Comment by connorjward** [comment, marked as answer]
Ah. This is a bug. The error should go away if you add `"netgen": false` to the `"options"` bit of `$VIRTUAL_ENV/.configuration.json`. I will get a proper fix in ASAP.

**Reply by yianzeng** [reply]
I'll give this a try! I'm struggling to find the file, where can I find .configuration.json?

**Reply by connorjward** [reply]
It's in the top level `firedrake` folder (the one containing `bin`, `src`, etc).

**Reply by yianzeng** [reply]
Seems to be working thank you !
